R/fotmob_players.R

```diff
@@ -102,7 +103,7 @@ fotmob_get_match_players <- function(match_ids) {
           dplyr::select(
             -.data$name
           ) %>%
-          dplyr::filter(!is.na(.data$value)) %>%
+          dplyr::filter(stringr::str_detect(.data$col, "^stats_"), !is.na(.data$value)) %>%
```

Comment:
there is a `title` column (pivoted to rows) that I want to drop here. In case fotmob changes what that column is named, I think it's better to select (filter since the columns are temporarily pivoted to rows) all other columns instead of dropping `title` explicitly (i.e. `filter(col != "title")`)